src/main/java/svc/data/municipal/CourtDAO.java

```diff
@@ -16,17 +16,11 @@
 @Repository
 public class CourtDAO extends BaseJdbcDao {
 	public Court getByCourtId(Long courtId) {
-		try {
```

Comment:
This try/catch may not need to go here, but it should be at another level - maybe the Manager could catch and re-throw it as a NotFoundException or something?